Jellyfin.Api/Helpers/BandwidthLimitOption.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+
+namespace Jellyfin.Api.Helpers;
+
+/// <summary>
+/// Options object for storing the download limit value.
+/// </summary>
+public class BandwidthLimitOption
+{
+    /// <summary>
+    /// Gets or sets the Guid of the users id.
+    /// </summary>
+    public Guid User { get; set; }
+
+    /// <summary>
+    /// Gets or sets the bandwidth per second for a download.
```

Comment:
In what unit is this bandwith expected?

Reply:
bytes. Same behavior as the Transcoding limit

Reply:
Updated code comments to clarify unit